test/extended/testdata/build-extended/bc-scripts-by-url.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "BuildConfig",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "java-extended-build-from-url"
+  },
+  "spec": {
+    "source": {
+      "git": {
+        "uri": "https://github.com/php-coder/java-maven-hello-world.git"
+      }
+    },
+    "strategy": {
+      "sourceStrategy": {
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "wildfly:10.0",
+          "namespace": "openshift"
+        },
+        "runtimeImage": {
+          "kind": "DockerImage",
+          "name": "phpcoder/base-jdk:8"
```

Comment:
@bparees @PI-Victor I'm using my custom image here, because I need to change `USER` from `jboss` to `1000`. Without this change OpenShift will reject it because of `AllowedUserIds` rule.